mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -521,13 +521,30 @@ func resourceContainerCluster() *schema.Resource {
 				Description: ` Description of the cluster.`,
 			},
 
-			"enable_binary_authorization": {
-				Default:     false,
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Description: `Enable Binary Authorization for this cluster. If enabled, all container images will be validated by Google Binary Authorization.`,
-				ConflictsWith: []string{"enable_autopilot"},
-			},
+                        "binary_authorization": {
+                                Type:         schema.TypeList,
+                                Optional:     true,
+                                MaxItems:     1,
+                                Description:  "Configuration options for the Binary Authorization feature.",
```

Comment:
We can't support the change as is.

This would be a breaking change for our customers.

Reply:
So the changes need to be strictly additive?

Reply:
correct, otherwise existing customers would be unable to consume the new provider without a diff. 

We could add the new field with a deprecation on the old one